contracts/mirror_gov/src/staking.rs

```diff
@@ -299,9 +328,16 @@ fn withdraw_user_voting_rewards<S: Storage>(
     storage: &mut S,
     user_address: &CanonicalAddr,
     token_manager: &TokenManager,
-) -> u128 {
-    let w_polls: Vec<(Poll, VoterInfo)> =
-        get_withdrawable_polls(storage, token_manager, user_address);
+    poll_id: Option<u64>,
+) -> StdResult<(u128, Vec<u64>)> {
+    let w_polls: Vec<(Poll, VoterInfo)> = match poll_id {
+        Some(poll_id) => {
+            let poll: Poll = poll_read(storage).load(&poll_id.to_be_bytes())?;
+            let voter_info = poll_voter_read(storage, poll_id).load(&user_address.as_slice())?;
+            vec![(poll, voter_info)]
```

Comment:
Don't we need to conduct withdrawable check?

Reply:
like https://github.com/Mirror-Protocol/mirror-contracts/pull/62/files#diff-b4800aa15fd9828e1b90abbf8f4d797616137eb2e7701b8134759542e6833ad3L337

Reply:
right. Fixed it